src/UI/Movies/Delete/DeleteMovieView.js

```diff
@@ -21,7 +21,7 @@ module.exports = Marionette.ItemView.extend({
         var deleteFiles = this.ui.deleteFiles.prop('checked');
         var addExclusion = this.ui.addExclusion.prop('checked');
         this.ui.indicator.show();
-        
+        this.model.set('deleted', true); 
```

Comment:
Why do this? `this.model.destroy` should set 'deleted' on the model itself, shouldn't it?